docker/.env

```diff
@@ -0,0 +1,29 @@
+# host ip
+HOST_IP=1.2.3.4 # you must be adjust 1.2.3.4 to your host IP.
+
+# CouchDB 
+COUCHDB_PORT=5984
+COUCHDB_USER=admin
+COUCHDB_PASSWORD=password
+COUCHDB_DB=ehsm_kms_db
+
+# docker image name
+IMAGE_NAME_EHSM_BASE=ehsm_base
+
+# eHSM version
+vsn_major=0
+vsn_minor=2
+vsn_patch=0
+EHSM_VERSION_STR=${vsn_major}.${vsn_minor}.${vsn_patch}

```

Comment:
is there some way to get this version automatically?

Reply:
About the version,We need to discuss it .